docs/profilers/tag_prof.md

```diff
@@ -30,15 +30,42 @@ That's how a report looks like:
 
 ## Instructions
 
-TagProf can only be used with RSpec.
+TagProf can be used with both RSpec and Minitest.
 
 To activate TagProf use `TAG_PROF` environment variable:
 
+With Rspec:
 ```sh
 # Group by type
 TAG_PROF=type rspec
 ```
 
+With Minitest:
+```sh
+# using pure ruby
+TAG_PROF=type ruby 
+
+# using Rails built-in task
+TAG_PROF=type bin/rails test
+```
+
+### Usage specificity with Minitest
+
+Minitest does not support the usage of tags by default. TagProf therefore groups statistics by direct subdirectories of the root test directory. It assumes root test directory is named either `spec` or `test`.
+
+When no root test directory can be found the test statistics will not be grouped with other tests. They will be displayed per test with a significant warning message in the report.
```

Comment:
I think, we should use the `__unknown__` value for unrecognized tests (that's how we deal with RSpec specs without defined tags).

Reply:
Also, I think, we should make it clear in the docs that if some other value beyound "type" is provided, it's ignored silently.